### PR TITLE
Update ROADMAP.md to include data connector statuses

### PR DESCRIPTION
**Pull request by digadeesh**

### CODE REVIEW

File: docs/ROADMAP.md [hunk 1]
```diff
@@ -8,42 +8,82 @@ If you have a feature request or suggestion, please [get in touch](https://githu
...
+- Databricks data connector (Stable)
+- Spice AI data connector (Stable)
+- S3 data connector (Stable)
+- DuckDB data connector (Stable)
+- ODBC Data Connector (Stable)
+- FTP/SFTP Data Connector (Stable)
+- PostgreSQL Data Connector (Stable)
+- Snowflake Data Connector (Beta)
+
+## v1.0-stable Data Connector Support
```
Comment (lukekim) [inline docs/ROADMAP.md]: h3

File: docs/ROADMAP.md [hunk 2]
```diff
@@ -8,42 +8,82 @@ If you have a feature request or suggestion, please [get in touch](https://githu
...
+  | ------------- | ------------------- |
+  | `databricks`  | Stable              |
+  | `postgres`    | Stable              |
+  | `spiceai`     | Stable              |
+  | `s3`          | Stable              |
+  | `dremio`      | Alpha               |
+  | `mysql`       | Alpha               |
+  | `duckdb`      | Stable              |
+  | `clickhouse`  | Alpha               |
+  | `odbc`        | Stable              |
```
Comment (lukekim) [inline docs/ROADMAP.md]: sort table stable to alpha

Comment (digadeesh) [reply docs/ROADMAP.md]: done

File: docs/ROADMAP.md (hunk 1 above)
Comment (digadeesh) [reply docs/ROADMAP.md]: done

**Review by lukekim — APPROVED** [review]

**Pull request merged**